makefiles/py.inc.mk

```diff
@@ -0,0 +1,19 @@
+# generate .py.h header files of .py files
+PY_HEADER_DIR ?= $(BINDIR)/$(MODULE)py_h
+PY = $(wildcard *.py)
```

Comment:
Specifying the python modules manually is currently not possible, e.g., as "SRC_PY". The reason is that the application is built with "applications.mk", which overrides any application specific variables.

Consider the fail case (user script gets processed): a header will be generated in a non-clashing folder. IMO, that's acceptable.

Or do you have an example where a user had a python script in their application folder, and creating a header would break stuff?

This can be revisited once we have a more generic blob -> header solution.